idaes/surrogate/pysmo/sampling.py

```diff
@@ -533,21 +627,23 @@ class UniformSampling(SamplingMethods):
 
     """
 
-    def __init__(self, data_input, list_of_samples_per_variable, sampling_type=None, edges=None):
+    def __init__(self, data_input, list_of_samples_per_variable, sampling_type=None, xlabels=None, ylabels=None, edges=None):
         """
         Initialization of UniformSampling class. Three inputs are required.
 
         Args:
             data_input (NumPy Array, Pandas Dataframe or list) :  The input data set or range to be sampled.
 
-                - When the aim is to select a set of samples from an existing dataset, the dataset must be a NumPy Array or a Pandas Dataframe and **sampling_type** option must be set to "selection". The output variable (Y) is assumed to be supplied in the last column.
-                - When the aim is to generate a set of samples from a data range, the dataset must be a list containing two lists of equal lengths which contain the variable bounds and **sampling_type** option must be set to "creation". It is assumed that no range contains no output variable information in this case.
+                - When the aim is to select a set of samples from an existing dataset, the dataset must be a NumPy Array or a Pandas Dataframe and **sampling_type** option must be set to "selection". A single output variable (y) is assumed to be supplied in the last column if **xlabels** and **ylabels** are not supplied.
```

Comment:
This seems like cut and paste of the comment in the other class. Suggest you just refer to it, or some superclass comment, instead of duplicating the text (same comment for other duplications of this block)

Reply:
I prefer duplicating comments because I want the user to be able to use (and read about) any sampling class without having to navigate elsewhere for instructions.